packages/lib-classifier/src/store/WorkflowStepStore.js

```diff
@@ -47,6 +47,27 @@ const WorkflowStepStore = types
       }
 
       return false
+    },
+
+    findTasksByType (type) {
+      const tasksByType = Array.from(self.steps).map(([stepKey, step]) => {
+        if (step?.tasks) {
+          return Object.values(step.tasks).filter(task => {
+            return task.type === type
+          })
+        }
+        
+        return []
+      })
+
+      return tasksByType.flat()
+    },
+
+    get interactionTask () {
```

Comment:
Can we name this `markingTask`? I find the generic name confusing because all tasks involve some sort of interaction (choosing an answer, making a mark etc.)

Reply:
It's `interactionTask` because it's being used by the `InteractionLayer`.